cts/cts-cli.in

```diff
@@ -407,22 +407,21 @@ class Test:
                 rc = abs(rc)
                 rc_str = signal.strsignal(rc)
             else:
-                rc = ExitStatus(rc)
-                rc_str = str(rc)
+                rc_str = str(ExitStatus(rc))
```

Comment:
This doesn't leave `rc` in the same state as previously, but I'm pretty sure it's fine since we're no longer using the `%d` format specifier to cast it back from `ExitStatus`.

Reply:
> This is actually a surprisingly tricky little block of code that functions differently on different OSes.

I'm not sure what you mean here. But this change no longer seems to be present and I don't even know what commit it was in.